src/vue-wrapper.ts

```diff
@@ -78,6 +79,13 @@ export class VueWrapper implements WrapperAPI {
     return Array.from(results).map((x) => new DOMWrapper(x))
   }
 
+  setData(data: Record<string, any>) {
+    // lodash.merge merges by *reference* so this will update
+    // any existing data with the newly passed data.
+    merge(this.componentVM.$data, data)
```

Comment:
Should we guard against setting data that is not in `$data` already? Not sure

Reply:
Let's check how Vue 3 behaves. I wonder if the new reactivity API allows you to update with new properties? I think we should match whatever Vue 3 does.